### DISCUSSION TITLE
Migration guide from `react-tsparticles`

### DISCUSSION THREAD

**Discussion by adamalston** [matteobruni/tsparticles#5300] [Q&A]
Hi, is there a migration guide from `react-tsparticles` to `@tsparticles/engine` and `@tsparticles/react`? I have searched though the issues and discussions in this repo and I haven't found one. I am trying to migrate https://github.com/adamalston/v2 (a Typescript project) which uses `react-tsparticles` version `1.43.1` to `@tsparticles/engine` and `@tsparticles/react`.

Particles component: https://github.com/adamalston/v2/blob/b317ab46c6392fb027a670610df5c6fddd7c8808/src/components/Particles.tsx
Particles config: https://github.com/adamalston/v2/blob/b317ab46c6392fb027a670610df5c6fddd7c8808/src/appearance/options.ts

**Comment by matteobruni** [comment]
There's no migrating guide from v1 to v2, you can find something useful here with the 2.0 changelog: https://dev.to/tsparticles/tsparticles-205-is-out-breaking-changes-ahead-keeping-the-1x-support-2n63
After the v2 migration, migrating to v3 syntax should be easy
